From: Jiri Slaby <jslaby@suse.cz>
Date: Tue, 29 Apr 2014 14:35:12 +0200
Subject: kabi: kgr, add reserved fields
Patch-mainline: never, kabi
References: fate#313296

Add some reserved fields to kGraft structures for future uses.

Signed-off-by: Jiri Slaby <jslaby@suse.cz>
[mmarek@suse.cz: Only patch task_struct]
---
 include/linux/sched.h |    2 ++
 1 file changed, 2 insertions(+)

--- a/include/linux/sched.h
+++ b/include/linux/sched.h
@@ -1480,6 +1480,8 @@ struct task_struct {
 	unsigned int	sequential_io;
 	unsigned int	sequential_io_avg;
 #endif
+	/* Reserved for kGraft */
+	unsigned long	suse_kabi_padding;
 };
 
 /* Future-safe accessor for struct task_struct's cpus_allowed. */
